pallets/gear/src/benchmarking/syscalls.rs

```diff
@@ -1064,6 +1064,35 @@ where
         Self::prepare_handle(code, 0)
     }
 
+    pub fn gr_error(r: u32) -> Result<Exec<T>, &'static str> {
+        let status_code_offset = 1;
+        let error_offset = status_code_offset + size_of::<i32>() as i32;
+        let error_len_offset = error_offset + size_of::<u32>() as i32;
```

Comment:
better go with

status_code_offset = 1;
error_len_offset = status_code_offset + size_of::\<i32> 
error_offset = error_len_offset + size_of::\<u32>

to avoid rewriting error len on writing error bytes itself